postcss.config.js

```diff
@@ -1,17 +1,6 @@
-const purgecss = require("@fullhuman/postcss-purgecss");
-
 module.exports = {
   plugins: [
-    purgecss({
-      content: ["./**/*.js"],
-      safelist: {
-        deep: [
-          /su-bg-[A-Za-z0-9]*(-)?([A-Za-z0-9]*)?/,
-          /su-text-[A-Za-z0-9]*(-)?([A-Za-z0-9]*)?/,
-        ],
```

Comment:
I think we will still want a safelist of colors as I don't think we have them all in our source at the moment and they are a common thing to use as input examples. Is the safelist working in JIT now?

Reply:
Yes, theoretically safelist should be working in JIT mode now after version 2.2
https://blog.tailwindcss.com/tailwindcss-2-2#built-in-safelist-transform-and-extract-support

Good point about the colors. Perhaps we can safe list the `su-bg-somecolor` and the `su-text-somecolor` classes 🤔

Reply:
Safelist has been moved to tailwind.config.js.